tests/test_memfs.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import warnings
+from concurrent.futures import ThreadPoolExecutor
+from unittest import TestCase, mock
+from uuid import uuid4
+
+from snekbox.memfs import MemFS
+
+UUID_TEST = uuid4()
+
+
+class MemFSTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.logger = logging.getLogger("snekbox.memfs")
+        self.logger.setLevel(logging.WARNING)
+        warnings.filterwarnings(
+            "ignore",
+            ".*Implicitly cleaning up.*",
+            ResourceWarning,
+            "snekbox.memfs",
+        )
```

Comment:
I think this filter may persist for other test cases. Is that a concern?

Reply:
This was due to the multithread test relying on implicit cleanup. I've since changed it to use contextlib.ExitStack instead to call `__exit__`, so this is not necessary and removed in [d33a55d](https://github.com/python-discord/snekbox/pull/159/commits/d33a55d775c9f15b6cd9089c7e2de3e827f0b837)